### PR TITLE
clang tidy naming for some type of  variables

### PR DESCRIPTION
**Pull request by Yaraslaut**
refs #960

### CODE REVIEW

File: src/crispy/boxed.h [hunk 1]
```diff
@@ -38,7 +38,7 @@ namespace helper
 // }}}
 
 template <typename T>
-constexpr bool is_boxed = helper::is_boxed<T>::value;
+constexpr bool IsBoxed = helper::is_boxed<T>::value;
```
Comment (christianparpart) [inline src/crispy/boxed.h:41]: boxed.h should retain it's C++/STL like naming conventions, maybe?

**Review by christianparpart — CHANGES_REQUESTED** [review]
only very few minors, I think. Overall looking good. Many thanks.

File: src/crispy/boxed.h (hunk 1 above)
Comment (Yaraslaut) [reply src/crispy/boxed.h:41]: this file will be deleted soon, no need to update PR

File: src/vtpty/UnixPty.cpp [hunk 2]
```diff
@@ -369,39 +367,6 @@ int UnixPty::write(std::string_view data)
...
-
-    if (select(nfds, &rfd, &wfd, &efd, &tv) < 0)
-        return -1;
-
-    if (!FD_ISSET(_masterFd, &wfd))
-    {
-        ptyOutLog()("PTY write of {} bytes timed out.\n", size);
-        return 0;
-    }
-
```
Comment (christianparpart) [inline src/vtpty/UnixPty.cpp]: this should never have compiled (`blocking` undefined, it was a function-param that was removed). How did this end up in this commit? 🤔

**Review by christianparpart — APPROVED** [review]
many thanks :)

**Pull request merged**